src/web/components/SignInGate/SignInGate.stories.tsx

```diff
@@ -25,6 +25,54 @@ export const mainStandalone = () => {
 };
 mainStandalone.story = { name: 'main_standalone' };
 
+export const mainStandaloneComment = () => {
+	return (
+		<Section>
+			<SignInGateMain
+				guUrl="https://theguardian.com"
+				signInUrl="https://profile.theguardian.com/"
+				dismissGate={() => {}}
+				ophanComponentId="test"
+				isComment={true}
+			/>
+		</Section>
+	);
+};
+mainStandaloneComment.story = { name: 'main_standalone_comment' };
```

Comment:
Not a huge deal but there's no need to use snake case for the `name` here, this can just be any display string. There's a sort of convention to have name like, 'with default` or 'when foobar is true' as then it reads nicely with the main Story name

Eg. 'My component - with defaults'